R/utils_accessibility.R

```diff
@@ -1,6 +1,6 @@
 #' h1_tabstop
 #'
-#' @param header Heading text
+#' @param heading Heading text
```

Comment:
I've changed all args that were `header` to `heading` throughout this file. The latter seems to have been what was intended, and make smore sense anyway.